pyscitt/pyscitt/governance.py

```diff
@@ -157,17 +157,6 @@ def set_scitt_configuration_proposal(configuration: dict) -> dict:
     }
 
 
-def set_ca_bundle_proposal(name: str, bundle: str) -> dict:
```

Comment:
Please update the demo scripts accordingly as we no longer need to set the x509 roots of trust for the instance configuration.

Reply:
I think the scripts themselves need updating to set a CWT claims and generate a policy. If we want the demo to continue to work, I suggest we find a way to hook it up to the CI, otherwise it's pretty much guaranteed to go stale.